cluster/src/main/java/com/linkedin/norbert/protos/NorbertExampleProtos.java

```diff
@@ -94,8 +94,10 @@ public int getSerializedSize() {
       return size;
     }
     
+    private static final long serialVersionUID = 0L;
```

Comment:
To be honest, I'm not really sure what serialver does here (and I couldn't get the serialver command to work). I changed it to 1L, which seems to be the default.